packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -318,14 +318,64 @@ class DateTimePicker extends React.Component {
 
   handleTimeChange(event, time) {
     this.timeValue = time;
-    const validDate = DateUtil.isValidDate(this.dateValue, this.state.dateFormat) && this.isDateTimeAcceptable(DateTimeUtils.convertDateTimeStringToMomentObject(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds, this.initialTimeZone));
+    let validDate = DateUtil.isValidDate(this.dateValue, this.state.dateFormat) && this.isDateTimeAcceptable(DateTimeUtils.convertDateTimeStringToMomentObject(this.dateValue, this.timeValue, this.state.dateFormat, this.props.showSeconds, this.initialTimeZone));
     const validTime = DateTimeUtils.isValidTime(this.timeValue, this.props.showSeconds);
-    const previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : null;
+    let previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : null;
+
+    if (event.type === 'keydown') {
+      // Keyboard shortcut is pressed, split the time value into hour and minute inputs
+      const timeArray = this.timeValue.split(':');
+      if (event.key === 'n' || event.key === 'N') {
+        // `N` set Date to today
+        previousDateTime = DateUtil.createSafeDate(DateUtil.getCurrentDate(), this.initialTimeZone);
+        validDate = true;
+      } else if (event.key === '-' || event.key === '_') {
+        // `-` subtract 1 minute`
+        if (!validDate) {
+          // Date is not valid, set date to today making it valid
+          previousDateTime = DateUtil.createSafeDate(DateUtil.getCurrentDate(), this.initialTimeZone);
+          validDate = true;
+        }
+
+        if (timeArray[0] === '23' && timeArray[1] === '59') {
+          // Time is 23:59 which means previous datetime should be on the wrong day, lower it by 1
+          previousDateTime = previousDateTime.subtract(1, 'd');
```

Comment:
See similar [comment](https://github.com/cerner/terra-framework/pull/1520/files#r820890748) below regarding subtracting a day that goes beyond the supported MIN date of 01/01/1900.

Reply:
Resolved in this comment https://github.com/cerner/terra-framework/pull/1520/files#r824176148

> **[@BenBoersma](https://github.com/BenBoersma)** --  [(Mar 10, 2022, 3:33 PM CST)](https://github.com/cerner/terra-framework/pull/1520/files#r824176148):
>
> Fixed in Date-picker here [ea6e6e8#diff-dafaabc64f600b92fbfa865adda17d41223fea5f885e15310c2e88582c58e0dcR502](https://github.com/cerner/terra-framework/commit/ea6e6e82248964185d5d81ae2f63f8e50f6603ed#diff-dafaabc64f600b92fbfa865adda17d41223fea5f885e15310c2e88582c58e0dcR502) along with adding tests for it
> 
> Currently still 100% correctly implemented in date-time-picker for when increasing or decreasing the minute causes us to go over. but tests for that have been added. [d798c38#diff-8f8488b16a2c0f0e4089aa01407d27d8ba3116e1f11e277b8d3554ad88972292R977](https://github.com/cerner/terra-framework/commit/d798c38e2975d58fd2abecc55bc87db4aed4ac68#diff-8f8488b16a2c0f0e4089aa01407d27d8ba3116e1f11e277b8d3554ad88972292R977)